courses/serializers.py

```diff
@@ -303,6 +303,14 @@ def get_certificate(self, enrollment):
         """
         Resolve a certificate for this enrollment if it exists
         """
+        # No need to include a certificate if there is no corresponding wagtail page
+        # to support the render
+        if (
+            not enrollment.run.course.page
+            or not enrollment.run.course.page.certificate_page
+        ):
+            return None
```

Comment:
seems perfect but do we have the corresponding test as well to make sure we are getting `None` in case of page absence ?

Reply:
Tested this manually, we can have the hotfix rollout and I can add automated test for that subsequently.